server/src/main/java/org/opensearch/rest/RestHandler.java

```diff
@@ -163,6 +164,14 @@ public ReplacedRoute(Method method, String path, Method deprecatedMethod, String
             this.deprecatedPath = deprecatedPath;
         }
 
+        public ReplacedRoute(Method method, String path, String deprecatedPath){
```

Comment:
Minor nit: Thanks for adding these constructors, they seem subtle differences, can you add java doc comments for all the 3 of them?

Reply:
Minor: add a space before `{`

Reply:
Sure, done